src/L1SharedBridge.sol

```diff
@@ -352,59 +224,42 @@ contract L1SharedBridge is IL1SharedBridge, ReentrancyGuard, Ownable2StepUpgrade
                 txDataHash: txDataHash
             });
         }
+
         emit BridgehubDepositInitiated({
             chainId: _chainId,
             txDataHash: txDataHash,
             from: _prevMsgSender,
             to: _l2Receiver,
             l1Token: _l1Token,
-            amount: amount
+            amount: _depositAmount
         });
     }
 
     /// @notice Confirms the acceptance of a transaction by the Mailbox, as part of the L2 transaction process within Bridgehub.
     /// This function is utilized by `requestL2TransactionTwoBridges` to validate the execution of a transaction.
-    function bridgehubConfirmL2Transaction(
-        uint256 _chainId,
-        bytes32 _txDataHash,
-        bytes32 _txHash
-    ) external override onlyBridgehub whenNotPaused {
-        require(depositHappened[_chainId][_txHash] == 0x00, "ShB tx hap");
+    function bridgehubConfirmL2Transaction(uint256 _chainId, bytes32 _txDataHash, bytes32 _txHash)
```

Comment:
I think a different linter is used, it's fine but it would be a bit better even for auditors to minimise the diff.

Reply:
Could be, I'm just using foundry's fmt